recipes/libx264/all/conanfile.py

```diff
@@ -99,7 +99,7 @@ def _configure_autotools(self):
 
         if self._with_nasm:
             # FIXME: get using user_build_info
-            self._override_env["AS"] = os.path.join(self.deps_cpp_info["nasm"].rootpath, "bin", "nasm{}".format(".exe" if tools.os_info.is_windows else "")).replace("\\", "/")
+            self._override_env["AS"] = os.path.join(self.dependencies.build["nasm"].package_folder, "bin", "nasm{}".format(".exe" if tools.os_info.is_windows else "")).replace("\\", "/")
```

Comment:
what is the min version of conan supporting `self.dependencie.build`?

Reply:
Thanks for looking it up for me. I stared at the code-base for 20 minutes and couldn't figure it out.